plutus-core/plutus-core/src/PlutusCore/Data.hs

```diff
@@ -150,7 +150,7 @@ encodeBs b | BS.length b <= 64 = CBOR.encodeBytes b
 -- It's a bit tricky to get cborg to emit an indefinite-length bytestring with chunks that we control,
 -- so we encode it manually
 -- See Note [Evading the 64-byte limit]
-encodeBs b                     = CBOR.encodeBytesIndef <> foldMap encode (to64ByteChunks b) <> CBOR.encodeBreak
+encodeBs b = CBOR.encodeBytesIndef <> foldMap encode (to64ByteChunks b) <> CBOR.encodeBreak
```

Comment:
Not sure how this happened.  I think my editor must have automatically applied stylish-haskell after I pressed the wrong key and then deleted the result while I was looking at this file.